docs/examples/custom_operations/custom_operator/naive_histogram/README.md

```diff
@@ -0,0 +1,48 @@
+# DALI custom operator example
+
+This is an example/template of a DALI Custom operator.
+The implemented operation is a (very) naive histogram on a grayscale image.
+The NaiveHistogram operator is a GPU-only operator, which uses a badly written
+CUDA kernel to calculate the histogram. The purpose here is not to show-case
+how the write CUDA kernels, but to present how to use CUDA kernels within
+DALI operators.
+
+# Running the example
+
+To run the example, please follow the steps:
+
+1. Clone or download the repo
+1. Install DALI wheel, for example using [these instructions](https://docs.nvidia.com/deeplearning/dali/main-user-guide/docs/installation.html#pip-official-releases)
+1. Build the `libnaivehistogram.so`:
```

Comment:
Shouldn't those be `1. 2. 3.`?

Reply:
Neeee, markdown handles the numbering and putting all of them as `1.` makes it easier to edit ;)

You can click "View file" to see the rendered outcome.

Reply:
There is `4.` and `5.` already in the source later.

Reply:
Yes, because there's a code snippet between, therefore I need to start from `4`

Reply:
:facep